code/game/gamemodes/steal_items.dm

```diff
@@ -6,8 +6,8 @@
 #define THEFT_FLAG_UNIQUE 2
 
 /datum/theft_objective
-	var/name = ""
-	var/typepath=/atom
+	var/name = "this objective is impossible, yell at a coder"
+	var/typepath=/obj/effect/debugging
```

Comment:
I did this so if someone created an "abstract" subtype it wouldn't cause cryopods to do this:

<img width="261" alt="dont_use_the_cryopods" src="https://user-images.githubusercontent.com/5661700/30000161-a21ad222-9016-11e7-933c-67e80c7e3b55.PNG">

Reply:
*Soylent green is people*

*The cryopods chop you up into little pieces*

*Wear this tinfoil hat so the Grey's can't read your mind*

Reply:
I mean we can't return after we logout. Makes sense to me